crates/driver/src/commit_reveal.rs

```diff
@@ -7,8 +9,12 @@ use solver::{
 
 /// A `SolutionSummary` holds all information solvers are willing to disclose during settlement
 /// competition. It does **not** have to include the call data, yet.
-#[derive(Debug, Clone, Deserialize, Serialize)]
-pub struct SettlementSummary {}
+#[derive(Debug, Default, Clone, Deserialize, Serialize)]
+pub struct SettlementSummary {
+    pub surplus: f64,
```

Comment:
Maybe we need to specify that the surplus is denominated in the native token (i.e. Ether/xDai).